src/codeflare_sdk/common/utils/unit_test_support.py

```diff
@@ -40,6 +40,11 @@ def create_cluster_config(num_workers=2, write_to_file=False):
         worker_memory_limits=6,
         appwrapper=True,
         write_to_file=write_to_file,
+        annotations={
+            "app.kubernetes.io/managed-by": "test-prefix",
+            "key1": "value1",
+            "key2": "value2",
+        },
```

Comment:
```suggestion
        annotations={
            "app.kubernetes.io/managed-by": "test-prefix",
            "key1": "value1",
            "key2": "value2",
        },
```

This is a helpful support function for creating a basic AppWrapper Ray Cluster for config specific tests/etc. The annotations should not be added here.